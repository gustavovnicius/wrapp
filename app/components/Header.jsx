import React from 'react';
import {
  Flex,
} from 'reflexbox';

export default function Header() {
  return (
    <Flex align="center" justify="center">
      <h1>Statistics</h1>
    </Flex>
  );
}
