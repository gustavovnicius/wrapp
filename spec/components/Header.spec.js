import React from 'react';
import renderer from 'react-test-renderer';
import Header from 'components/Header';

describe('Header', () => {
  it('match the snapshot', () => {
    const header = renderer.create(
      <Header />
    ).toJSON();

    expect(header).toMatchSnapshot();
  });
})
