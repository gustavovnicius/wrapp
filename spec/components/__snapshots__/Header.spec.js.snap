// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header match the snapshot 1`] = `
<div
  className="Flex"
  style={
    Object {
      "WebkitAlignItems": "center",
      "WebkitJustifyContent": "center",
      "alignItems": "center",
      "boxSizing": "border-box",
      "display": "flex",
      "justifyContent": "center",
      "msAlignItems": "center",
      "msJustifyContent": "center",
    }
  }
>
  <h1>
    Statistics
  </h1>
</div>
`;
